src/schema/objects/enums.yaml

```diff
@@ -1405,3 +1405,88 @@ microvascular:
   display_name: microvascular
   description: |
     The origin of a tissue: microvascular
+start_of_embryo_stage:
+  value: start of embryo stage
+  display_name: start of embryo stage
+  description: |
+    The beginning of the embryo stage, which is also commonly referred as the "conception".
+start_of_gestational_stage:
+  value: start of gestational stage
+  display_name: start of gestational stage
+  description: |
+    The beginning of the gestational stage, which is also commonly referred as the
+    "first day of the mother's last menstrual period".
+start_of_neonate_stage:
+  value: start of neonatal stage
+  display_name: start of neonatal stage
+  description: |
+    The beginning of the neonatal stage, which is also commonky referred as the "birth".
```

Comment:
```suggestion
    Point of birth marking the beginning of a subject's neonatal stage. Often assumed
    as default age reference if not otherwise specified.
```